### ISSUE TITLE
Down-arrow key not usable within ellipsis menu when navigating via keyboard

### ISSUE THREAD

**Issue by joelclimbsthings** [woocommerce/woocommerce-admin#6203]
**Describe the bug**
When navigating within a menu expanded via the `ellipsis-menu` component, the down arrow key is suppressed and does not function. This prevents any users from using the arrow keys to navigate the menu.

After looking into this briefly, I thought the logic that governs this _might_ be related to this function:
https://github.com/woocommerce/woocommerce-admin/blob/ec36a00b77b5c0e313985a5a64c88aaec9eb154e/packages/components/src/ellipsis-menu/menu-item.js#L31

However as far as I can tell this handler isn't called at all for the arrow keys (only other keys), despite the logic contained within the function to add behavior specifically for these key presses. I'm not certain why this handler is not being called but it might be related to this bug. 

**Bonus:**
I've also[ read that best practice](https://www.smashingmagazine.com/2017/11/building-accessible-menu-systems/) is for the tab keys to _not_ be utilized for navigating sub-menus, and to rely entirely on arrow keys for this purpose. This frees up the tab keys to escape the menu and return to the flow of the document. Potentially debatable. 

**To Reproduce**

- Go to WooCommerce > Home
- Navigate to ellipsis menu via the keyboard (click to the left of the menu, and hit tab until it's highlighted)
- Expand the menu with the `enter` key
- Attempt to navigate between the items using the arrow keys

**Screenshots**

![image](https://user-images.githubusercontent.com/444632/106060707-3af7f400-60a9-11eb-85fc-3eac95f3d602.png)

**Expected behavior**
Arrow keys should be supported for navigating these menu items.

**Comment by ilyasfoo** [comment]
I quickly tested, it seems arrow keys are intercepted somewhere else, as `onKeyDown` wasn't even fired when I pressed them, but other keys fire the method.

As this it'll take more effort to sleuth and fix, I'm closing this issue as it's not prioritized. 
p90Yrv-2Bc-p2
